### ISSUE TITLE
Possibility to select other lights than lights on entertainment zone

### ISSUE THREAD

**Issue by j0sk** [BradyBrenot/huestacean#104]
You can do it yourself HUE compatible light strips easily but those led strips cannot be added to the entertainment zone.  **So could this Huestacean modified that way that you can select other lamps as well?**

DIY hue led strips: https://char.gd/blog/2018/building-better-cheaper-philips-hue-led-strips

**Comment by BradyBrenot** [comment]
Yes, and I _might_ do it, but non-Entertainment-compatible lights are going to make for a fairly bad experience with Huestacean. The whole point of the Entertainment API is that you get extremely fast (low latency, high frequency) updates to light colors. Without that, well, try just tapping different colors in the Hue app and see how slow it is to transition between them.
